package/gluon-web-network/luasrc/lib/gluon/web/model/admin/network.lua

```diff
@@ -128,7 +128,14 @@ uci:foreach("system", "gpio_switch", function(s)
 		if not section then
 			section = f:section(Section)
 		end
-		local poe = section:option(Flag, s[".name"], translate("Enable " .. s.name))
+
+		local name, port = s.name:match("^(.+[^%d])(%d*)$")
```

Comment:
Please change this very generic match to one that explicitly matches `PoE Power Port(%d*)`.